src/apps/schedule/crud/events.py

```diff
@@ -121,12 +192,13 @@ async def save(self, schema: UserEventCreate) -> UserEvent:
         user_event: UserEvent = UserEvent.from_orm(instance)
         return user_event
 
-    async def get_by_event_id(self, event_id: int) -> list[int]:
+    async def get_by_event_id(self, event_id: int) -> int:
         """Return user event instances."""
-        query: Query = select(UserEventsSchema.user_id)
+        query: Query = select(distinct(UserEventsSchema.user_id))
         query = query.where(UserEventsSchema.event_id == event_id)
+        query = query.where(UserEventsSchema.is_deleted == False)  # noqa: E712
         result = await self._execute(query)
-        results: list[int] = result.scalars().all()
+        results: int = result.scalars().one_or_none()
```

Comment:
catch exception

Reply:
exception for None? I am expecting None as well, because of this model:
![image](https://user-images.githubusercontent.com/17030291/218742353-044cf3e3-226c-4272-be22-7fb687dc20e2.png)